src/urbit/app/chess.hoon

```diff
@@ -835,7 +838,7 @@
         games  (~(put by games) u.game-id [new-game *chess-position *(map @t @ud) | & | | | | |])
       ==
     ::
-    ::  XX: document this
+    ::  subscribe to updates on active games
```

Comment:
Not what this does: this handles frontends subscribing to the chess agent to receive updates so that they can visualize them